src/components/map/PermalinkFeaturesService.js

```diff
@@ -25,6 +25,12 @@ goog.require('ga_previewfeatures_service');
           var featureIdsByBodId = {};
           var paramKey;
           var listenerKey;
+          var forceZoom;
+
+          if (queryParams.zoom !== undefined && isFinite(queryParams.zoom)) {
+            forceZoom = parseInt(queryParams.zoom);
```

Comment:
Why do we need to force the zoom ?

Reply:
That is because I rebased on master and then mvt. Already in master